perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -93,10 +93,11 @@ sub bin_services_for_address {
         }
 
         push @site_services_filtered, {
-            id           => $service->{SiteServiceID},
-            service_id   => $service->{ServiceItemName},
-            service_name => $service->{ServiceItemDescription},
-            next         => {
+            id             => $service->{SiteServiceID},
+            service_id     => $service->{ServiceItemName},
+            service_name   => $service->{ServiceItemDescription},
```

Comment:
I think it might be better to use $self->{c}->stash->{containers} to give this a nice name - would just mean implementing it basically for now, but would mean when it's updated this would have the front facing names

Reply:
I am not entirely sure what you are asking here - could be because I am unclear what we are doing about `stash->{containers}` (I will need to ask on Slack, as I thought we had a ticket to handle {containers}, but it doesn't seem to exist. Currently it's just empty, marked with a '# TODO' - possibly because we don't have a hardcoded list of services (unlike what I can tell from other cobrands) but have to check dates on the fly).

Reply:
Understand what you mean now; have added a commit for containers :-)

Reply:
I saw you talking to Chris, an he answered more clearly than I would have managed!